app/graphql/types/activity_event_type.rb

```diff
@@ -2,5 +2,6 @@ module Types
   class ActivityEventType < Types::BaseEnum
     value "USER LOGGED IN", value: "user_logged_in"
     value "USER REGISTERED", value: "user_registered"
+    value "USER UPDATED PASSWORD", value: "user_reset_password"
```

Comment:
```suggestion
    value "USER RESET PASSWORD", value: "user_reset_password"
```

Should we have consistent names?